deploy/operator/ztp/deploy_spoke_cluster.sh

```diff
@@ -136,3 +136,4 @@ if [ ${SPOKE_CONTROLPLANE_AGENTS} -eq 1 ] ; then
 fi
 
 setup_libvirt_dns
+open_firewall_ports
```

Comment:
@paul-maidment out of curiosity, what job / flow did it fix?

Reply:
It doesn't exist yet https://github.com/openshift/release/pull/28958